draft-ietf-tls-esni.md

```diff
@@ -994,6 +966,72 @@ The "ech_is_inner" is expected to have an empty payload. If the payload is
 non-empty (i.e., the length of the "extension_data" field is non-zero) then the
 backend server MUST abort the handshake with an "illegal_parameter" alert.
 
+## Handling HelloRetryRequest {#server-hrr}
+
+The HelloRetryRequest constructed by the backend server is called the
+HelloRetryRequestInner message; and the HelloRetryRequest constructed by the
+client-facing server is called the HelloRetryRequestOuter message.
```

Comment:
This duplicates text above,

Reply:
Removed. I thought it would be useful to duplicate this text, but apparently not :)